### PR TITLE
feat: [CHAOS-1347]: Adding new nav links to chaos sidebar

### PR DESCRIPTION
**Pull request by SahilKr24**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by vanshBhatia-A4k9 — APPROVED** [review]

File: src/modules/75-chaos/components/ChaosSideNav/__test__/ChaosSideNav.test.tsx [hunk 1]
```diff
@@ -88,4 +89,42 @@ describe('Chaos Sidenav Render', () => {
...
+          accountId: ':accountId',
+          orgIdentifier: ':orgIdentifier',
+          projectIdentifier: ':projectIdentifier'
+        })}
+        pathParams={{ accountId: 'dummyAccID', orgIdentifier: 'dummyOrgID', projectIdentifier: 'dummyProjID' }}
+      >
+        <ChaosSideNav />
+      </TestWrapper>
+    )
+    expect(container).toMatchSnapshot()
```
Comment (knagurski) [inline src/modules/75-chaos/components/ChaosSideNav/__test__/ChaosSideNav.test.tsx:128]: Same here, you can shift the flag into the wrapper and replace the snapshot with a real test that proves the link shows.

File: src/modules/75-chaos/components/ChaosSideNav/__test__/ChaosSideNav.test.tsx [hunk 2]
```diff
@@ -88,4 +89,42 @@ describe('Chaos Sidenav Render', () => {
...
```
[the remark this answers is not in the document]
Comment (SahilKr24) [reply src/modules/75-chaos/components/ChaosSideNav/__test__/ChaosSideNav.test.tsx]: Updated PR to check for specific nav labels

File: src/modules/75-chaos/components/ChaosSideNav/__test__/ChaosSideNav.test.tsx [hunk 3]
```diff
@@ -88,4 +89,42 @@ describe('Chaos Sidenav Render', () => {
...
+        path={routes.toChaosMicroFrontend({
+          accountId: ':accountId',
+          orgIdentifier: ':orgIdentifier',
+          projectIdentifier: ':projectIdentifier'
+        })}
+        pathParams={{ accountId: 'dummyAccID', orgIdentifier: 'dummyOrgID', projectIdentifier: 'dummyProjID' }}
+      >
+        <ChaosSideNav />
+      </TestWrapper>
+    )
```
[the remark this answers is not in the document]
Comment (SahilKr24) [reply src/modules/75-chaos/components/ChaosSideNav/__test__/ChaosSideNav.test.tsx]: Didn't know this existed, updating the PR to use this prop. Thanks.

**Review by knagurski — APPROVED** [review]
Nice! Much better and more accurate 👍🏻

**Review by arkajyotiMukherjee — APPROVED** [review]

**Pull request merged**